airbyte-api/src/main/openapi/config.yaml

```diff
@@ -599,6 +599,29 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/sources/most_recent_source_actor_catalog:
```

Comment:
@benmoriceau FYI for when you are back.  We will need to ensure this gets ported into the Micronaut branch of airbyte-server